docs/AboutUs.adoc

```diff
@@ -4,53 +4,52 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 3 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
-{empty} +
+Powerlifting Competition Coordinator (PCC) is being developed by https://github.com/AY1920S1-CS2103-T11-3[T11-Team 3]. +
+
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
-== Project Team
+== PCC Project Team
 
-=== John Doe
+=== Prof. Damith C. Rajapakse
 image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]]
 
 Role: Project Advisor
 
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Ho Wei Chin
+image::HoWeiChin.png[width="180", align="left"]
+{empty}[https://github.com/HoWeiChin[github]] [<<howeichin#, portfolio>>]
 
 Role: Team Lead +
-Responsibilities: UI
+Responsibilities: Deliverable, Tracking
 
 '''
 
-=== Johnny Doe
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== Ooi Ming Sheng
+image::ooimingsheng.png[width="180", align="left"]
+{empty}[https://github.com/ooimingsheng[github]] [<<ooimingsheng#, portfolio>>]
```

Comment:
for some reason, our images are not showing up in the rich diff mode. can check on the reason. Will it appear once we merge?

Reply:
yes i assume it will show once the PR is merged and the images are imported.